### DISCUSSION TITLE
Parallel run scenarios into one specs

### DISCUSSION THREAD

**Discussion by xraze666** [getgauge/gauge#2426] [General]
Hello!
I have multiple specification file and each specification contains multiple scenarios. Is it possible to run multiple scenarios in parallel across specification file and specification run Consistently?

**Comment by lirany1** [comment]
https://docs.gauge.org/execution.html?os=linux&language=java&ide=vscode#parallel-execution-1

we use the gradle plugin with its params
https://github.com/getgauge/gauge-gradle-plugin#all-additional-properties
